src/main/java/AN_hw_4/Task3.java

```diff
@@ -0,0 +1,17 @@
+package AN_hw_4;
+
+public class Task3 {
+    public static void main(String[] args) {
+        String text = "The forest sheds its crimson attire,\nSilver frost wilts the field,\nThe day peeks in as if by chance\nAnd hides behind the surrounding mountains.";
+        System.out.println(text);
+        text = text.toLowerCase();
+        String[] words = text.split(" ");
+        int count = 0;
+        for (String word : words) {
+            if (word.equals("the")) {
+                count++;
+            }
+        }
+        System.out.println("The word 'the' occurs " + count + " times in the text.");
```

Comment:
I can see four words 'the'
![Screenshot_15](https://github.com/AntBelyi/ALevel_automation/assets/66726924/80a5bec8-ccf0-4b23-a6d9-4a73573cba2d)

But your output is:
![Screenshot_16](https://github.com/AntBelyi/ALevel_automation/assets/66726924/5a7d6126-984b-47f2-8bd5-eeb7b0aed92f)

Try to implement a Scanner. Whenever substring i type into - your program should count how much entered from console substrings in your text

Reply:
fixed